util/BUILD.bazel

```diff
@@ -0,0 +1,7 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+exports_files([
```

Comment:
Add a comment:

TODO: make this a proper binary resource with `py_binary`.

Reply:
Done. Tracked in #41.